docs/hu/docs/index.md

```diff
@@ -0,0 +1,470 @@
+```
+
+<details markdown="1">
+<summary>Vagy használd az <code>async def</code>-et...</summary>
+
+Ha a kódod `async` / `await`-et, használ `async def`:
+
+```Python hl_lines="9  14"
+from typing import Union
+
+from fastapi import FastAPI
+
+app = FastAPI()
+
+
+@app.get("/")
+async def read_root():
+    return {"Hello": "World"}
+
+
+@app.get("/items/{item_id}")
+async def read_item(item_id: int, q: Union[str, None] = None):
+    return {"item_id": item_id, "q": q}
+```
+
+**Megjegyzés**:
+
+Ha nem tudod, tekintsd meg a
+If you don't know, check the _"Sietsz?"_ szekciót <a href="https://fastapi.tiangolo.com/async/#in-a-hurry" target="_blank">`async` és `await`-ről dokumentációba</a>.
+
+</details>
+
+### Futtasd le
+
+Indítsd el a szervert a következő paranccsal:
+
+<div class="termy">
+
+```console
+$ uvicorn main:app --reload
+
+INFO:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
+INFO:     Started reloader process [28720]
+INFO:     Started server process [28722]
+INFO:     Waiting for application startup.
+INFO:     Application startup complete.
+```
+
+</div>
+
+<details markdown="1">
+<summary>A parancsról <code>uvicorn main:app --reload</code>...</summary>
+
+A `uvicorn main:app` parancs a következőre utal:
+
+* `main`: fájl `main.py` (a Python "modul").
+* `app`: a `main.py`-ban a `app = FastAPI()` sorral létrehozott objektum.
+* `--reload`: kód változtatás esetén újra indítja a szervert. Csak fejlesztés közben használandó.
+
+</details>
+
+### Ellenőrizd
+
+Nyisd meg a böngésződ a következő címen: <a href="http://127.0.0.1:8000/items/5?q=somequery" class="external-link" target="_blank">http://127.0.0.1:8000/items/5?q=somequery</a>.
+
+A következő JSON választ fogod látni:
+
+```JSON
+{"item_id": 5, "q": "somequery"}
+```
+
+Máris létrehoztál egy API-t ami:
+
+* HTTP kéréseket fogad a  `/` és `/items/{item_id}` _útvonalakon_.
+* Mindkét _útvonal_ a `GET` <em>műveletet</em> használja (másik elnevezés: HTTP _metódus_).
+* A `/items/{item_id}` _útvonalnak_ van egy _path paramétere_, az `item_id`, aminek `int` típusúnak kell lennie.
+* A `/items/{item_id}` _útvonalnak_ még van egy opcionális, `str` típusú _query paramétere_ is, a `q`.
+
+### Interaktív API dokumentáció
+
+Most nyisd meg a <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a> címet.
+
+Az automatikus interaktív API dokumentációt fogod látni (amit a <a href="https://github.com/swagger-api/swagger-ui" class="external-link" target="_blank">Swagger UI</a>-al hozunk létre):
+
+![Swagger UI](https://fastapi.tiangolo.com/img/index/index-01-swagger-ui-simple.png)
+
+### Alternatív API dokumentáció
+
+És most menj el a <a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a> címre.
+
+Az alternatív automatikus dokumentációt fogod látni. (provided by <a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank">ReDoc</a>):
```

Comment:
I don't have special cases right now here, but maybe "la'sd" instead of "provided by"?